### ISSUE TITLE
[PvP] Deep Thunder procs way too often

### ISSUE THREAD

**Issue by vvranceanu** [chromiecraft/chromiecraft#6052]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

https://www.wowhead.com/tbc/item=28441/deep-thunder procs too often. It has streaks of procs of 2-3 times in under 15 seconds. I had 2 instances of arena matched i was stunned at first touch and the stuns kept comming for me and my arena teammate. we were stunned 5-7 times per person per match, conservatively.
I was stunned in 2 instances twice in a bladestorm(one on video) and it was generally such a frequent occurence in arena vs deep thunder + healer(it gave us ample time to observe, unless dps-dps that overlaps these procs with stuns of their own) that it made me realize something is for sure off.

### Expected Blizzlike Behaviour

For a weapon speed of 3.8, with a documented proc rate of 1ppm, there should be a proc per hit of 3.8 * 1/60 = 0.063%. Thats just not how 1% works(please check videos)

https://filetransfer.io/data-package/5oKKXUgi#link

### Source

https://www.wowdb.com/items/28441-deep-thunder
https://wowwiki-archive.fandom.com/wiki/Procs_per_minute
https://www.wowhead.com/tbc/item=28441/deep-thunder#comments
https://www.reddit.com/r/classicwow/comments/o9mvso/question_about_thunderdeep_thunder_on_ret/
https://youtu.be/bPhcAhmedT4?t=474

### Steps to reproduce the problem

1. Duel a deep thunder wielder, standard dps rotation

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/86bfef8874883a1726966326368918567c5531cd

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by vvranceanu** [comment]
Can you please have a look at this? I get 3 games per week of arena and in 4 of them i sit stunned locked by this...

**Comment by Maria-sequel** [comment]
![image](https://github.com/chromiecraft/chromiecraft/assets/20357406/22191fd7-969f-43b2-b12a-f6f24c5bc54b)

**Comment by Salehyn** [comment]
Ok, so I can confirm this is bugged. I made the following notes:

The CC database currently has a ppm=1 value, which translates to roughly 6,3% (0.063=6.3%) proc chance. This is the value corresponding to patch 2.1-2.2 on TBC, when this weapon lived its golden age after being buffed, and before being nerfed. This is no longer the correct value for 3.3.5a.
From wowhead: "Not only does DT have incredible stats and damge... but it gives a 5% chance to stun on ANY melee hit." - **this is from patch 2.1.2** https://www.wowhead.com/wotlk/item=28441/deep-thunder#comments
**Meanwhile this is a comment from 3.0.2** (after nerfs) "The proc has a VERY low chance, around 2%. Still very good stats and dps, also very slow so its indeed a fantastic pala weapon, but dont expect your judge damage to be much better just because you'll get some lucky procs once every 10-20 fights."

Tests made on the CC PTR confirm this 6%ish value is currently active. With purely auto attacks,the observed proc rate was 5.9%, over a 13minute test sample, for a total of 14 procs (~1.07ppm)
Tests made on the WOTLK Classic realms of blizzard (performed by Bench) show a proc chance of merely 1.5% (15 procs over 1000 hits) in the same conditions (purely auto attacks against a level 70 target)

This confirms the nerf to "around 2%" suggested by the wowhead user on 3.0.2.
1.5% proc chance translates to roughly 0,24 PPM, this is the observed value for 3.3.5a, as opposed to the 1ppm in DB. (roughly 1/4 of the current proc chance)
Furthermore, tests made on the WOTLK classic realms of Blizzard (performed by Bench) show a proc chance of merely 0,8% against a **level 80** target. It is safe to say the proc chance is greatly diminished against level 80 targets.
On CC the proc chance against level 80 seems to be the same as against level 70.

Further notes:
The database ppm only applies to the base weapon attack frequency and that is intended. https://wowwiki-archive.fandom.com/wiki/Procs_per_minute
Haste buffs and any special attacks that use the melee weapon will add extra procs, so in reality, a 1ppm value in the database translates to more than 1 proc per minute when using special attacks.

Testing on CC PTR with special attacks (Windfury and stormstrike) added, the observed proc rate remained around 6%, however this translated to 2,6 ppm (26 procs over 10mins) as the frequency of attacks greatly increased. This frequently led to chain procs to the point the target became immune through diminishing returns. This is what players on the live realm currently experience.

-------------------------------------------------------------------------------
Bottom line: The current 1ppm value (6,3%) should be reduced to about 0.24 (1,5%) when used against level 70 targets, and diminished to about 0,12 (0,8%) when used against level 80 targets.

**Comment by Maria-sequel** [comment]
anything ppm bug related should be a new issue
